src/test/java/net/sf/marineapi/nmea/parser/RMCTest.java

```diff
@@ -23,7 +24,7 @@
 public class RMCTest {
 
 	/** Example sentence */
-	public static final String EXAMPLE = "$GPRMC,120044.567,A,6011.552,N,02501.941,E,000.0,360.0,160705,006.1,E,A*0B";
+	public static final String EXAMPLE = "$GPRMC,120044.567,A,6011.552,N,02501.941,E,000.0,360.0,160705,006.1,E,A,S*74";
```

Comment:
Maybe add another legacy example and some tests for the 12-field version as well?